nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/ChartProcessor.java

```diff
@@ -998,8 +1025,8 @@ boolean scriptHasDataset(String script) {
 
     private void setDatasetFile(String datasetName) {
         File scriptDir = getScriptDir();
-        datasetFile = scriptDir.toPath().resolve(datasetName + extension).toFile();
-        datasetFileTemp = scriptDir.toPath().resolve(datasetName + extension + ".tmp").toFile();
+        datasetFile = scriptDir.toPath().resolve(datasetName + datasetType).toFile();
```

Comment:
I've removed the use of datsetFileTemp.  It was previously used because sometimes memory mapped files on Windows couldn't be deleted. So I would write to a tmp file and then copy to final name.  That code is now changed so the use of .tmp files shouldn't be necessary.